common/src/main/java/io/apicurio/registry/rest/v2/beans/SearchedArtifact.java

```diff
@@ -85,6 +89,12 @@
     @JsonProperty("labels")
     @JsonPropertyDescription("")
     private List<String> labels = new ArrayList<String>();
+    /**
+     *
+     */
+    @JsonProperty("properties")
+    @JsonPropertyDescription("")
+    private Map<String,String> properties = new HashMap<String,String>();
```

Comment:
Good change, just note that we need to update the OpenAPI description as well.  But don't worry about that, I will need to do that (because we actually maintain the OAI description in Apicurio Studio and then generate the JAX-RS from there).